### PR TITLE
Update test cns daemonset to reconcile its own state

### PR DESCRIPTION
**Pull request by camrynl**
**Reason for Change**:

enabling ManageEndpointState in configmap and updating cns daemonset in e2e tests so that cns will reconcile its own state

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

File: test/integration/manifests/cns/daemonset.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ spec:
           effect: NoSchedule
       containers:
         - name: cns-container
-          image: acnpublic.azurecr.io/azure-cns:v1.4.32
+          image: acnpublic.azurecr.io/azure-cns:v1.4.33
```
Comment (tamilmani1989) [inline test/integration/manifests/cns/daemonset.yaml:44]: lets use latest v1.4.35

**Review by tamilmani1989 — CHANGES_REQUESTED** [review]

File: test/integration/manifests/cns/daemonset.yaml (hunk 1 above)
Comment (rbtr) [reply test/integration/manifests/cns/daemonset.yaml:44]: this is just a placeholder, the tests update the image to the latest working version on the branch

**Review by tamilmani1989 — APPROVED** [review]

**Review by rbtr — CHANGES_REQUESTED** [review]
we still need coverage of the reconcile-from-CNI scenario, I don't think we can update this for all swift E2Es

**Comment by camrynl** [comment]
> we still need coverage of the reconcile-from-CNI scenario, I don't think we can update this for all swift E2Es

If I use different configmaps for cilium and swift, would that keep coverage for reconciling from cni?

**Review by tamilmani1989 — APPROVED** [review]
lgtm

**Review by rbtr — APPROVED** [review]

**Pull request merged**